definitions/npm/react-helmet_v6.x.x/test_react-helmet.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import Helmet from 'react-helmet';
+const HelmetComponent = () => (
+  <Helmet
+    htmlAttributes={{"lang": "en", "amp": undefined}} // amp takes no value
+    title="My Title"
+    titleTemplate="MySite.com - %s"
+    defaultTitle="My Default Title"
+    base={{"target": "_blank", "href": "http://mysite.com/"}}
+    meta={[
+      {"name": "description", "content": "Helmet application"},
+      {"property": "og:type", "content": "article"}
+    ]}
+    link={[
+      {"rel": "canonical", "href": "http://mysite.com/example"},
+      {"rel": "apple-touch-icon", "href": "http://mysite.com/img/apple-touch-icon-57x57.png"},
+      {"rel": "apple-touch-icon", "sizes": "72x72", "href": "http://mysite.com/img/apple-touch-icon-72x72.png"}
+    ]}
+    script={[
+      {"src": "http://include.com/pathtojs.js", "type": "text/javascript"},
+      {"type": "application/ld+json", innerHTML: `{ "@context": "http://schema.org" }`}
+    ]}
+    onChangeClientState={(newState) => console.log(newState)}
+  />
+)
+
+
+const fail = () => (
+    // $FlowExpectedError
+    <Helmet title={{}} />
+)
+
+// rewind and renderStatic are aliases
+const heads = [Helmet.rewind(), Helmet.renderStatic()];
+
+heads.forEach((head) => {
+  (head.htmlAttributes.toString(): string);
+  (head.title.toString(): string);
+  (head.base.toString(): string);
+  (head.meta.toString(): string);
+  (head.link.toString(): string);
+  (head.script.toString(): string);
+  (head.style.toString(): string);
+
+  // $FlowExpectedError
```

Comment:
If you're adding tests here can you please add suppression codes https://flow.org/en/docs/errors/#toc-making-suppressions-more-granular-with-error-codes

Reply:
@Brianzchen I totally stand by the Flow types as I believe I know what I am doing there!

Insofar as test go... I literally copied and pasted them from the ones in v5

I didn't know they weren't gonna work

I am totally inexperienced at Flow tests

Do we need them for this?

Is it better if I just remove that and stick to the types?

Reply:
so in your local I assume you're on macos you can run `./quick_run_def_tests.sh` and you'll see a bunch of errors show up. It's much easier to review the code also if all the suppression codes are fixed. There's not many it seems in this lib so if you're ok to fix that that would be appreciated.